source/layouts/layout.slim

```diff
@@ -12,6 +12,7 @@ html
     - categories.each do |category|
       = feed_tag :atom, "#{config[:platform_url]}/categories/#{category.slug}.xml", title: "#{config[:platform_title]} – #{category.title} articles"
     = stylesheet_link_tag :blog
+    script src="//platform-api.sharethis.com/js/sharethis.js#property=58ea34c2b23bb10011d6ce9a&product=inline-share-buttons" type="text/javascript"
```

Comment:
@skatkov good point, thanks – I've blindly pasted the script as sharethis requested, but I'm sure it is possible to optimize loading a bit.